recovery/cmd/backup_size_test.go

```diff
@@ -0,0 +1,121 @@
+package cmd_test
+
+import (
+	"testing"
+
+	"github.com/openshift-kni/cluster-group-upgrades-operator/recovery/cmd"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCompare(t *testing.T) {
+
+	testcases := []struct {
+		estimated, freeDisk float64
+		expected            bool
+		name                string
+	}{
+		{
+			estimated: 80.00 * 1024 * 1024 * 1024,
```

Comment:
nit: Can you add comments to each of these to explicitly call out the units/sizes for each test?

Reply:
Added by this commit https://github.com/openshift-kni/cluster-group-upgrades-operator/commit/0a94f2debff8eb3ed881ec1eab12431501f50218 as suggested